README.md

```diff
@@ -0,0 +1,11 @@
+libvmemcache
+============
+
+### pmem.io/vmemcache web site
+
+This repo contains documentation for libvmemcache.
+
+To view documentation go to: [manpages](http://pmem.io/vmemcache/vmemcache.3.md)
```

Comment:
This path is wrong (on pmdk-convert it is wrong too)

Reply:
Done